.github/gbp.toml

```diff
@@ -2,19 +2,16 @@ no_balance_label = "GBP: No Update"
 reset_label = "GBP: Reset"
 
 [points]
-"Atomic" = 2
```

Comment:
why get rid of atomic?

Reply:
We don't use it.

Reply:
we do use it

Reply:
we used it in 34 PRs out of 38,000 PRs

no, we don't use it

Reply:
it's a new damn label

Reply:
do u know how long its existed